scripts/unpack_benchmarks.sh

```diff
@@ -14,10 +14,14 @@ unlzma_mv_untar () {
   if [[ "$TAR_FILE.lzma" -nt "$TAR_FILE"  ]] ; then
     unlzma -k "$TAR_FILE.lzma"
   fi
-  if [[ "$OSTYPE" == "darwin"* ]]; then
+  TARVER=`tar --version | head -n 1`
+  if [[ "$TARVER" =~ "tar (GNU tar)" ]]; then
+    tar --overwrite -xf $TAR_FILE
```

Comment:
Per our discussion elsewhere, the `--overwrite` flag might be better left off so this script can be rerun. (That would also match the bsdtar behavior, which uses the `-k    Keep (don't overwrite) existing files` flag.)